### PR TITLE
Remove usages of strings in queries

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Related issue: ballerina-platform/ballerina-standard-library#1880

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Module.md [hunk 1]
```diff
@@ -168,8 +168,9 @@ The `CREATE` statement is executed via the `execute` remote function of the clie
 
 ```ballerina
```
Comment (niveathika) [inline ballerina/Module.md:169]: Module.md and client.bal documentation is updated in sql module. Shall we do the same changes here too?

Comment (kaneeldias) [reply ballerina/Module.md:164]: Fixed in 43f2fe4

File: ballerina/Module.md [hunk 2]
```diff
@@ -103,9 +107,7 @@ connection pool handling.  For its properties and possible values, see the [`sql
 
    E.g., The DB client creation for an H2 database is as follows.
    ```ballerina
-    jdbc:Client|sql:Error dbClient = 
-                               new ("jdbc:h2:~/path/to/database", 
-                                "root", "root");
+    jdbc:Client|sql:Error dbClient = new ("jdbc:h2:~/path/to/database", "root", "root");
```
Comment (niveathika) [inline ballerina/Module.md:110]: This was formatted as such because code snippets in API docs have a line length restriction. Could you verify this?

Comment (kaneeldias) [reply ballerina/Module.md]: Adjusted in 2049fed

**Review by niveathika — APPROVED** [review]

**Pull request merged**